lib/types/timetable.ts

```diff
@@ -12,7 +12,7 @@ export type DevelopmentPlan = {
   developmentPlanGeography: string;
   documentationUrl: string;
   adoptedDate: string;
-  organisations: string[];
+  organisations: string;
```

Comment:
I have made this field a string rather than string[] for now as it makes it easier to parse the csv string and I wasn't sure how to go about it if it was an array as we parse the csv by separating by ",". Any suggestions?

Reply:
I think we're OK keeping with string for now